.github/workflows/build-test.yaml

```
@@ -448,6 +453,8 @@ jobs:
           chmod +x bin/*
       - uses: ./.github/actions/kots-e2e
         with:
+          # REVIEW: missing required `kots-namespace` parameter. Set to empty string`
+          kots-namespace: ''
```

I believe this should be:
```suggestion
          kots-namespace: 'qakotsregression'
```
Based on https://github.com/replicatedhq/kots/blob/main/e2e/testim/inventory/inventory.go#L28-L30